static/src/javascripts/lib/time-utils.spec.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import { daysSince } from 'lib/time-utils';
+
+describe('Days since', () => {
+    it('should return a correct duration give a valid date string', () => {
+        const correctDateString = '2017-06-08T15:26:14Z';
+        expect(daysSince(correctDateString)).not.toBe(Infinity);
```

Comment:
I'd mock `Date` so that we can assert the actual value here rather than just not being `Infinity`